src/jaxsim/physics/algos/soft_contacts.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
```

Comment:
I need this every time the following pattern is used:

```python
class MyClass:
    @staticmethod
    def build() -> MyClass:
        pass
``` 

It allows to use `-> MyClass` instead of `-> "MyClass"`. I discovered it not long time ago :)